docs/dev-machine-setup.md

```diff
@@ -0,0 +1,109 @@
+     ```
+     $ curl -fsSL -o ./firecracker https://github.com/firecracker-microvm/firecracker/releases/download/v0.11.0/firecracker-v0.11.0
+     $ chmod +x ./firecracker
+     $ curl -fsSL -o ./hello-vmlinux.bin https://s3.amazonaws.com/spec.ccfc.min/img/hello/kernel/hello-vmlinux.bin
+     $ curl -fsSL -o ./hello-rootfs.ext4 https://s3.amazonaws.com/spec.ccfc.min/img/hello/fsfiles/hello-rootfs.ext4
+     ```
+  1. Run Firecracker
+     1. Run the VMM (if everything is working, it will NOT return to your command prompt):
+        ```
+        $ rm -f /tmp/firecracker.sock && ./firecracker --api-sock /tmp/firecracker.sock
+        ```
+     1. Open an additional terminal, SSH into your `firecracker-vm`. Configure and start the Guest VM (ignore the 'No Content' outputs):
+        ```
+        $ curl --unix-socket /tmp/firecracker.sock -i \
+          -X PUT 'http://localhost/boot-source' -H 'Accept: application/json' -H 'Content-Type: application/json' \
+          -d '{"kernel_image_path":"./hello-vmlinux.bin","boot_args":"console=ttyS0 reboot=k panic=1 pci=off"}'
+        $ curl --unix-socket /tmp/firecracker.sock -i \
+          -X PUT 'http://localhost/drives/rootfs' -H 'Accept: application/json' -H 'Content-Type: application/json' \
+          -d '{"drive_id":"rootfs","path_on_host":"./hello-rootfs.ext4","is_root_device":true,"is_read_only":false}'
+        $ curl --unix-socket /tmp/firecracker.sock -i \
+          -X PUT 'http://localhost/actions' -H  'Accept: application/json' -H  'Content-Type: application/json' \
+          -d '{"action_type":"InstanceStart"}'
+        ```
+     1. Switch back to the **first** terminal (where VMM is running), use 'root/root' to log into the guest.
+        ```
+        Welcome to Alpine Linux 3.8
+        Kernel 4.14.55-84.37.amzn2.x86_64 on an x86_64 (ttyS0)
+        
+        localhost login: root
+        Password:
+        Welcome to Alpine!
+        
+        The Alpine Wiki contains a large amount of how-to guides and general
+        information about administrating Alpine systems.
+        See <http://wiki.alpinelinux.org>.
+        
+        You can setup the system with the command: setup-alpine
+        
+        You may change this message by editing /etc/motd.
+        
+        login[858]: root login on 'ttyS0'
+        localhost:~#
+        ```
+        **Done!**
+
+# Appendix
+## Setting up a Google Cloud Platform account
+Google Compute Engine (GCE) supports nested KVM virtualization on a variety of VM types, and hence could be a convenient choice for Firecracker development. Furthermore, if you create a GCP account, you can get $300 credits for a 1-year trial - which should be more than enough for Firetracker development for a while.
```

Comment:
Don't refer to GCP credit policies in a Firecracker document.

Reply:
Yup I think this is out of scope for this doc (again, imagine it with 4-5 sets of instructions, it ends up being a market survey of free tiers in all clouds 😄). It's also hard to maintain as these offers keep changing every few months (across a wider selection of clouds).